internal/core/data/messaging/eventpublisher.go

```diff
@@ -22,25 +24,25 @@ import (
 const (
 	ZEROMQ int = iota
```

Comment:
Do we need these constants to be capitalized? This is not Go idiomatic. `zmq` will fork fine here.

Reply:
Removed these constants altogether due to Fede's comment above and subsequent rework.